### PR TITLE
Activity stream

### LINKED ISSUE #925: code - gitcoin.co/activity
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a user, I want gitcoin.co/activity to show the recent activity on the platform, so I can see how active it is and follow the action.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: 
Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
no activity feed

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
activity feed that lists 

* opened issue
* started work
* stopped work
* submitted work
* accepted work
* increaed funding
* canceled bounty
* new tip
* tip received

items on gitcoin.co/activity

this will be used during livestream and also potentially asynchronously somewhere else.. it could live at gitcoin.co/activity

please see #notif-gitcoin on gitcoin.coslack for inspiriation

##### Recommended backend architecture

New django app - activityfeed

New DB model -- `ActivityFeedItem`, inserted in dashboard/notifications [example of how the slack activity items are created now](https://github.com/gitcoinco/web/blob/master/app/dashboard/notifications.py#L156)

* type -- varchar, one of the above
* related_profile (foreign key to profile)
* related_item (foreign key to either a tip or a bounty)
* payload (JSON)

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

* an activity feed is programmed into gitcoin.co/activity
* the activity feed uses the referenced designs
* responsive designs used (tablet, mobile, desktop)
* pagination works
* backend DB is used to store the infromation in a way thats queryable in a performant way

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)
Designs 
https://github.com/gitcoinco/web/issues/804#issuecomment-382916898
39023093-ea7adc9a-43f5-11e8-9d59-aafb01fa76ea

### PR DESCRIPTION
**Pull request by darkdarkdragon**
WIP to review
fixes #925

### CODE REVIEW

File: app/app/urls.py [hunk 1]
```diff
@@ -170,6 +170,8 @@
     url(r'^mission/?', retail.views.mission, name='mission'),
     re_path(r'^results/?(?P<keyword>.*)/?', retail.views.results, name='results_by_keyword'),
     re_path(r'^results/?', retail.views.results, name='results'),
+    url(r'^activity/?', retail.views.activity, name='activity'),
+    url(r'^mission/?', retail.views.mission, name='mission'),
```
Comment (owocki) [inline app/app/urls.py]: mission is in here twice now

File: app/dashboard/tokens.py [hunk 2]
```diff
@@ -200,3 +200,10 @@ def addr_to_token(addr):
         if(token['addr'].lower() == addr.lower()):
             return token
     return False
+
+
+def token_by_name(name):
+    for token in tokens:
```
Comment (mbeacom) [inline app/dashboard/tokens.py]: `tokens` is undefined in this scope.

Comment (mbeacom) [reply app/dashboard/tokens.py]: I pushed a fix as soon as it popped on rollbar the other day? 🤔 https://github.com/gitcoinco/web/commit/2325aa7aa867fc7df29f3088172fc4a00a4cb513

Comment (owocki) [reply app/dashboard/tokens.py]: thx; i think i must have fixed another 500 issue on /activity then

**Pull request merged**